src/Hit/Git/Status.hs

```diff
@@ -67,7 +70,7 @@ data DiffName = DiffName
     }
 
 parseDiffName :: [Text] -> Maybe DiffName
-parseDiffName [t, name] = DiffName name <$> parsePatchType t
+parseDiffName (t : xs)  = DiffName (unwords xs) <$> parsePatchType t
```

Comment:
Could you also please add an example here that explains why this `unwords` is required? I see that later `formatRename` function uses `T.intercalate "->"`, so without having the format of `git` output in front of the eyes it's hard to tell what going on here. 

The documentation for `parseDiffStat` is great! And I think that the similar documentation can be written for `parseDiffName` (I'm not sure why I didn't do this, it seems very useful).